client/src/components/main/restoCard.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+function RestoCard(props) {
```

Comment:
```suggestion
function RestaurantCard({imgUrl, restoName, description}) {
```
rename component name and pass props using destructuring